tests/basic/kubevirt_test.go

```diff
@@ -827,3 +830,86 @@ var _ = Describe("{KubeVirtPvcAndPoolExpandWithAutopilot}", func() {
 		}
 	})
 })
+
+var _ = Describe("{RebootRootDiskAttachedNode}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("RebootRootDiskAttachedNode", "Reboot the node where VMs root disk is attached", nil, 0)
+		DeployVMTemplatesAndValidate()
+	})
+	var appCtxs []*scheduler.Context
+
+	itLog := "Reboot node where Kubevirt VMs root disk is attached"
+	It(itLog, func() {
+		appList := Inst().AppList
+		defer func() {
+			Inst().AppList = appList
+		}()
+		Inst().AppList = []string{"kubevirt-cirros-live-migration", "kubevirt-windows-vm"}
+		stepLog := "schedule a kubevirtVM"
+		Step(stepLog, func() {
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				log.Infof("RK=> Deploying iteration: %d", i+1)
+				appCtxs = append(appCtxs, ScheduleApplications("reboot")...)
+			}
+		})
+		//defer DestroyApps(appCtxs, nil)
+		ValidateApplications(appCtxs)
```

Comment:
does this check if the VM is ready?  Otherwise, we will need to add that.

Reply:
yes, it does check it via schedOps.